src/main/java/org/myrobotlab/service/GoogleSearch.java

```diff
@@ -94,7 +92,8 @@ public void clearFilters() {
 
   @Override
   public void setLocale(String code) {
-    locale = new Locale(code);
+    // locale = new Locale(code);
+    Runtime.getInstance().setLocale(code);

```

Comment:
You should be able to set the language of your search.  This should not be setting the locale of runtime.

Reply:
been completely removed and pushed up into Service